src/test/java/liquibase/ext/mongodb/lockservice/ReplaceChangeLogLockStatementIT.java

```diff
@@ -0,0 +1,98 @@
+package liquibase.ext.mongodb.lockservice;
+
+/*-
```

Comment:
This disclaimer is not required in new files, it was used by the initial contributors but we don't use it anymore.